apps/barista/src/app.module.ts

```diff
@@ -103,6 +105,8 @@ import { DtToastModule } from '@dynatrace/barista-components/toast';
     BaToc,
     BaSidenav,
     BaScrollToTop,
+    BaColor,
```

Comment:
Please also add both components to the `BA_CONTENT_COMPONENTS` in `apps/barista/src/components` – then we can use it on all content pages instead of the html code we currently have one some Strapi pages :)

Reply:
you can remove the `BaColor` component here in this file if you add it to the `BA_CONTENT_COMPONENTS` array as @lara-aigmueller said